source/MaterialXGenShader/HwShaderGenerator.h

```diff
@@ -350,6 +350,9 @@ class HwShaderGenerator : public ShaderGenerator
     virtual void emitEdfNodes(const ShaderGraph& graph, const ShaderNode& shaderNode, HwClosureContextPtr ccx,
                               GenContext& context, ShaderStage& stage, string& edf) const;
 
+    /// Emit code for active light count definition
+    virtual void emitActiveLightSourceDefinition(GenContext& context, ShaderStage& stage) const;
```

Comment:
Nitpick: This method is not emitting code, it's creating a uniform that is needed by lighting. Code for this is emitted later among code for all other uniforms created. So I wonder if we should rename the method to something like `addStageLightingUniforms()`. It would also generalize the usage so we can add in other uniforms for lighting here later, if needed. `addStageLightingUniforms` would also be consistent with the existing method `addStageUniform` (called inside) which is the most general method for adding uniforms.